layouts/partials/recruiter.html

```diff
@@ -0,0 +1,34 @@
+<!-- <script type="text/javascript" src="//downloads.mailchimp.com/js/signup-forms/popup/embed.js" data-dojo-config="usePlainJson: true, isDebug: false"></script> -->
+<!-- <script type="text/javascript">require(["mojo/signup-forms/Loader"], function(L) { L.start({"baseUrl":"mc.us9.list-manage.com","uuid":"6f1977de9eff4c384dc8d6527","lid":"f030c0e09f"}) })</script> -->
+
+
+<!-- Begin MailChimp Signup Form -->
+<!-- <link href="//cdn-images.mailchimp.com/embedcode/horizontal-slim-10_7.css" rel="stylesheet" type="text/css"> -->
+
+<div id="mc_embed_signup_research">
+  <a class="handle-button">Help us improve cloud.gov</a>
+
+  <div class="form-container">
+    <p>
+      We’re looking for volunteers to participate in a 30-minute phone research interview about cloud.gov. If selected, we’ll contact you to set up a time at your convenience.
+    </p>
+    <form action="https://gsa.us9.list-manage.com/subscribe/post?u=6f1977de9eff4c384dc8d6527&amp;id=f030c0e09f" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" class="validate" target="_blank" novalidate>
+      <div id="mc_embed_signup_scroll">
+        <input type="email" value="" name="EMAIL" class="email" id="mce-EMAIL-research" placeholder="email address" required>
+        <!-- real people should not fill this in and expect good things - do not remove this or risk form bot signups-->
```

Comment:
This is somewhat alarming to have in our source code. Can it be removed or rephrased?

Reply:
That's been in the Mailchimp signup code since we added it a couple years ago - https://github.com/18F/cg-landing/pull/89/files - I think it's ok, it's pretty normal to have anti-spam measures in code.